src/main/java/com/scalar/db/storage/multistorage/MultiStorage.java

```diff
@@ -0,0 +1,158 @@
+package com.scalar.db.storage.multistorage;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.scalar.db.api.Delete;
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.Get;
+import com.scalar.db.api.Mutation;
+import com.scalar.db.api.Operation;
+import com.scalar.db.api.Put;
+import com.scalar.db.api.Result;
+import com.scalar.db.api.Scan;
+import com.scalar.db.api.Scanner;
+import com.scalar.db.exception.storage.ExecutionException;
+import com.scalar.db.service.StorageModule;
+import com.scalar.db.storage.common.util.Utility;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.concurrent.ThreadSafe;
+import javax.inject.Inject;
+
+/**
+ * A storage implementation with multi-storage for {@link DistributedStorage}.
+ *
+ * <p>This storage implementation holds multiple storage instances and have mapping a table name to
```

Comment:
```suggestion
 * <p>This storage implementation holds multiple storage instances. It chooses a storage instance on the basis of the specified configuration and a given operation. 
```

Reply:
I feel it's OK but it can be extended by giving examples if it's too blur.
Also, the following lines can be deleted if the above suggestion is accepted.